zerver/views/webhooks/hellosign.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from six.moves import range
+from typing import Dict, Any, Optional, Union
+
+def format_body(signatories, model_payload):
+    # type: (List[Dict[str, Any]], Dict[str, Any]) -> Any
+    RESULT_TEMPLATE = "The {contract_title}"
```

Comment:
This should be `result_template`; according to pep8, only global constants should be in uppercase. This is not a constant even.